crypto/src/main/scala/org/bitcoins/crypto/FieldElement.scala

```diff
@@ -40,5 +40,5 @@ object FieldElement
   private val G: ECPublicKey = ECPublicKey(
```

Comment:
Where does this `G` come from? Since a field isn't a group, does it have its own generator? If so, what field is this a part of ?

Reply:
So actually it is the same, I was just using decompressed in one spot and compressed here, decompressed is more efficient so I've changed this to use that as well.

Reply:
In fact, now that this is in `CryptoParams`, I've just delegated to there and deleted this `val` altogether